src/App.jsx

```diff
@@ -1,5 +1,11 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
+import { Button } from './components/Button';
+import { Goods } from './components/Goods/Goods';
+
+const SORT_BY_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
```

Comment:
if you're familiar with `enum` you can use it. https://www.typescriptlang.org/docs/handbook/enums.html#string-enums

Or just use an object instead
```
  const sortType = {
    alphabet: 'alphabet',
    length: 'length',
    default: '',
  }
```
But let's work with `enum`
```
enum SortType {
  alphabet = 'alphabet',
  length = 'length',
  default = '',
}
```

Reply:
@Viktor-Kost Sorry I can't use enum, I changed in package.json projectType, but It didn't work(
get this message -> File '@mate-academy/students-ts-config' not found.